routing/router.go

```diff
@@ -483,12 +483,9 @@ func (r *ChannelRouter) Start() error {
 
 	// If AssumeChannelValid is present, then we won't rely on pruning
 	// channels from the graph based on their spentness, but whether they
-	// are considered zombies or not.
-	if r.cfg.AssumeChannelValid {
-		if err := r.pruneZombieChans(); err != nil {
```

Comment:
Alternatively, couldn't we just launch it in a goroutine?

Reply:
that essentially what this is achieving. The reason I figured it made sense to delay it by 30 sec was to achieve less contention on the db while lnd is starting up.